Winget-AutoUpdate/functions/Start-NotifTask.ps1

```diff
@@ -14,6 +14,13 @@ function Start-NotifTask {
         [Switch]$ButtonDismiss = $false
     )
 
+    $caller = Get-ChildItem $MyInvocation.PSCommandPath | Select-Object -Expand Name
+    if ($caller -eq "User-Run.ps1") {
+        $Title = "Winget-AutoUpdate (WAU)"
+        $Balise = "Winget-AutoUpdate (WAU)"
+        $UserRun = $True
+    }
+
```

Comment:
and what about adding default values in the param ? something like :

```
function Start-NotifTask {
    param(
        [String]$Title = "Winget-AutoUpdate",
        [String]$Message,
        [String]$MessageType,
        [String]$Balise = "WAU",
        [String]$OnClickAction,
        [String]$Body,
        [String]$Button1Text,
        [String]$Button1Action,
        [Switch]$ButtonDismiss = $false
        [Switch]$UserRun = $false
    )
```

this way, `Title` and `Balise` got default values, and you just need to add `-UserRun` when you call this function from the `user-run.ps1` script

Reply:
Better